### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by nolanmar511**
Release notes: https://github.com/googleapis/cloud-profiler-nodejs/releases/tag/untagged-fd15418d529965aed686

### CODE REVIEW

**Comment by ofrobots** [comment]
I've edited the draft to this: https://github.com/googleapis/cloud-profiler-nodejs/releases/edit/untagged-75f39efe991e1b33c9d3. Hopefully this is easier for users to consume.

The part about using `node-pre-gyp` seemed like an implementation details that users probably don't care about. I took that out.

**Comment by aalexand** [comment]
I added

> The source maps support is on by default, specify `disableSourceMaps: true` in the profiler configuration to disable it.

sentence to the source maps feature summary.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/profiler",
-  "version": "0.2.3",
+  "version": "1.0.0",
   "description": "Adds support for Stackdriver Profiler to node.js applications",
```
Comment (aalexand) [inline package.json]:
```suggestion
  "description": "Adds support for Stackdriver Profiler to Node.js applications",
```

**Comment by aalexand** [comment]
Looks good to me, merge and release?

**Pull request merged**